packages/world/mud.config.ts

```diff
@@ -99,6 +99,26 @@ export default mudConfig({
       },
       tableIdArgument: true,
     },
+    KeysInTable: {
+      directory: "/modules/keysintable/tables",
+      primaryKeys: {},
+      schema: "bytes32[]", // For now only supports 1 key per value,
+      tableIdArgument: true,
+    },
+    KeysInTableLength: {
+      directory: "/modules/keysintable/tables",
+      primaryKeys: {},
+      schema: "uint256",
+      tableIdArgument: true,
+    },
+    UsedKeysIndex: {
+      directory: "/modules/keysintable/tables",
+      primaryKeys: {
+        keysHash: "bytes32",
+      },
+      schema: { has: "bool", index: "uint256" },
```

Comment:
`uint32` should be enough, MUD2 arrays are limited anyways and this will fit both cols into 1 slot